src/app/material.module.ts

```diff
@@ -0,0 +1,42 @@
+import {NgModule} from '@angular/core';
+import {
+  MdButtonModule,
+  MdCardModule,
+  MdDialogModule,
+  MdIconModule,
+  MdInputModule,
+  MdMenuModule,
+  MdOptionModule,
+  MdSidenavModule,
+  MdSlideToggleModule,
+  MdToolbarModule
+} from '@angular/material';
+
+@NgModule({
+  imports: [
+    MdButtonModule,
+    MdCardModule,
+    MdDialogModule,
+    MdIconModule,
+    MdInputModule,
+    MdMenuModule,
+    MdOptionModule,
+    MdSidenavModule,
+    MdSlideToggleModule,
+    MdToolbarModule
+  ],
+  exports: [
+    MdButtonModule,
+    MdCardModule,
+    MdDialogModule,
+    MdIconModule,
+    MdInputModule,
+    MdMenuModule,
+    MdOptionModule,
```

Comment:
I removed `MdOptionModule` and got:

```
zone.js:643 Unhandled Promise rejection: Template parse errors:
Can't bind to 'value' since it isn't a known property of 'md-option'.
1. If 'md-option' is an Angular component and it has 'value' input, then verify that it is part of this module.
2. If 'md-option' is a Web Component then add 'CUSTOM_ELEMENTS_SCHEMA' to the '@NgModule.schemas' of this component to suppress this message.
3. To allow any property add 'NO_ERRORS_SCHEMA' to the '@NgModule.schemas' of this component. ("r" [(ngModel)]="selectedAvatar">
        <md-option *ngFor="let avatar of avatars; let i = index;" [ERROR ->][value]="avatar">Avatar - {{i + 1}}</md-option>
      </md-select>
```